src/main/resources/locale_ru_RU.properties

```
@@ -242,7 +243,7 @@ listener.reaction.removal=Репутация выданная %DONOR% была 
 message.welcome=Спасибо за приглашение.\nВы можете начать настройку через команду /setup.
 messages.destruction=Это сообщение самоудалится через %MIN% минут(ы).
 selfCleanup.leave=Вы не настроили меня.\nЯ остаюсь только на активных серверах, в которых есть хотя бы один канал для репутации. Если хотите использовать меня вновь - можете в любой момент пригласить меня по этой ссылке [link](%INVITE%).
-selfCleanup.leave.title=Пока! o/
+selfCleanup.leave.title=Текстовые команды больше не поддерживаются. Вместо них используй команды со слешем.
```

This is the translation from `error.textCommand` (Suggesting revert)
```suggestion
selfCleanup.leave.title=Пока! o/
```